src/Test.sol

```diff
@@ -448,8 +449,471 @@ abstract contract Test is DSTest, Script {
+        AccessList[] accessList;
+        bytes data;
+        address from;
+        bytes gas;
+        bytes nonce;
+        address to;
+        bytes txType;
+        bytes value;
+    }
+
+    struct EIP1559Transaction {
+        string[] arguments;
+        address contractAddress;
+        string contractName;
+        string functionName;
+        bytes32 hash;
+        string opcode;
+        EIP1559TransactionDetail detail;
+    }
+
+    struct EIP1559TransactionDetail {
+        AccessList[] accessList;
+        bytes data;
+        address from;
+        uint256 gas;
+        uint256 nonce;
+        address to;
+        uint256 txType;
+        uint256 value;
+    }
+
+
+    struct LegacyTransaction {
+        string[] arguments;
+        address contractAddress;
+        string contractName;
+        string functionName;
+        string hash;
+        string opcode;
+        LegacyTransactionDetail transaction;
+    }
+
+    struct LegacyTransactionDetail {
+        AccessList[] accessList;
+        uint256 chainId;
+        bytes data;
+        address from;
+        uint256 gas;
+        uint256 gasPrice;
+        bytes32 hash;
+        uint256 nonce;
+        bytes1 opcode;
+        bytes32 r;
+        bytes32 s;
+        uint256 txType;
+        address to;
+        uint8 v;
+        uint256 value;
+    }
+
+    struct AccessList{
+        address accessAddress;
+        bytes32[] storageKeys;
+    }
+
+    struct RawReceipt {
+        bytes32 blockHash;
+        bytes blockNumber;
+        address contractAddress;
+        bytes cumulativeGasUsed;
+        bytes effectiveGasPrice;
+        address from;
+        bytes gasUsed;
+        RawReceiptLog[] logs;
+        bytes logsBloom;
+        bytes status;
+        address to;
+        bytes32 transactionHash;
+        bytes transactionIndex;
+    }
+
+    struct Receipt {
+        bytes32 blockHash;
+        uint256 blockNumber;
+        address contractAddress;
+        uint256 cumulativeGasUsed;
+        uint256 effectiveGasPrice;
+        address from;
+        uint256 gasUsed;
+        ReceiptLog[] logs;
+        bytes logsBloom;
+        uint256 status;
+        address to;
+        bytes32 transactionHash;
+        uint256 transactionIndex;
+    }
+
+    struct RawReceiptLog {
+        // json value = address
+        address logAddress;
+        bytes32 blockHash;
+        bytes blockNumber;
+        bytes data;
+        bytes logIndex;
+        bool removed;
+        bytes32[] topics;
+        bytes32 transactionHash;
+        bytes transactionIndex;
+        bytes transactionLogIndex;
+    }
+
+    struct ReceiptLog {
+        // json value = address
+        address logAddress;
+        bytes32 blockHash;
+        uint256 blockNumber;
+        bytes data;
+        uint256 logIndex;
+        bytes32[] topics;
+        uint256 transactionIndex;
+        uint256 transactionLogIndex;
+        bool removed;
+    }
+
+    struct TransactionReturn {
```

Comment:
When is this type used? In general might be helpful to help a short comment above each struct/function explaining where it's from / when to use it